src/main/java/seedu/address/logic/commands/DeleteEventCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+
+/**
+ * Deletes an event identified using its displayed index from the events list.
+ */
+public class DeleteEventCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete-event";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the event identified by the index number used in the displayed event list.\n"
+            + "Parameter: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Event:\n%1$s";
```

Comment:
Hmm you mean `MESSAGE_DELETE_EVENT_SUCCESS`?

Reply:
Good catch! Fixed